api/1.0/2020/04.json

```diff
@@ -30,12 +30,12 @@
       "host": "Eva Zils, online-recruiting.net",
       "length": "48",
       "size": "unlimited",
-      "travel": "no, virtual",
-      "prize": "yes",
+      "travel": "no",
+      "prize": "no",
       "highSchoolers": "yes",
       "cost": "free/donation",
-      "instagramURL": "https://twitter.com/HR_Hackathon",
-      "telegramURL": "https://hrhackathon.net",
+      "instagramURL": "https://www.instagram.com/ezils/",
+      "telegramURL": "https://twitter.com/HR_Hackathon",
```

Comment:
Can you change this to `twitterURL` instead of `telegramURL`? It's a bit misleading when you have that field with Twitter URL?  Also please fix the merge conflicts from our master branch so that I can safely merge it for you.